### PR TITLE
[RFC] vim-patch:7.4.1913

### PR DESCRIPTION
**Pull request by jbradaric**
```
vim-patch:7.4.1913
Problem:    When ":doautocmd" is used modelines are used even when no
            autocommands were executed. (Daniel Hahler)
Solution:   Skip processing modelines. (closes vim/vim#854)
```

https://github.com/vim/vim/commit/1610d052413e0ed664498853a47acc2d677a22d1

### CODE REVIEW

**Comment by brcolow** [comment]
Interestingly, this was ported differently from the upstream vim patch in that the `did_something = FALSE;`
was ported as `*did_something = false;` which is actually the correct way, and is indeed later corrected in patch [7.4.1928](https://github.com/vim/vim/commit/76ae22fef3cb224ca7fbf97517f881e825d4d0c2). This patch (1928) also corrects two typos which are not present in the neovim code base.

In other words, we can either mark 1928 as applied or N/A, but we should choose one so that it is no longer listed as not merged.

**Comment by justinmk** [comment]
Let's mark it as applied

**Comment by brcolow** [comment]
Done: https://github.com/neovim/neovim/pull/5653
